doc/src/site/sphinx/ml/sw_deep_learning.rst

```diff
@@ -40,7 +48,24 @@ and :ref:`model_details_H2ODeepLearningMOJOModel`.
         .. code:: scala
 
             import ai.h2o.sparkling.ml.algos.H2ODeepLearning
-            val estimator = new H2ODeepLearning().setLabelCol("CAPSULE").setHidden(Array(3, 2))
+            val estimator = new H2ODeepLearning()
+                               .setDistribution("tweedie")
+                               .setHidden(Array(1))
```

Comment:
The algorithm is configured with lots of parameters, but does it provide intended result with the network is simple? (Just one hidden layer of one node)